tests/acceptance/NewUser.feature

```diff
@@ -43,3 +42,19 @@ Feature: NewUser
     And I fill in the field "User_verifyCode" with "shazam"
     And I press the button "Register"
     Then I should see "Welcome!"
+
+  Scenario: Providing erroneous captcha prevents submission and show error message
```

Comment:
Does this scenario need an `@ok` tag in order for it to be run?

Reply:
@pli888, thanks for spotting, yes it should have a \@ok tag. Will amend accordingly